packages/fxa-settings/src/components/ConnectAnotherDevicePromo/en-US.ftl

```diff
@@ -9,6 +9,6 @@ connect-another-find-fx-mobile = Find { -brand-firefox } in the { -google-play }
 connect-another-play-store-image =
     .title = Download { -brand-firefox } on { -google-play }
 connect-another-app-store-image =
-    .title = Download { -brand-firefox } on { -app-store }
+    .title = Download { -brand-firefox } on the { -app-store }
```

Comment:
You need a new id for this.  Simply appending `-2` works.  (Without a new id translators wouldn't know about the change.)